SPECS/mesa-libGLU/mesa-libGLU.spec

```diff
@@ -0,0 +1,130 @@
+Summary:        Mesa libGLU library
+Name:           mesa-libGLU
+Version:        9.0.1
+Release:        4%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://mesa3d.org/
+Source0:        https://mesa.freedesktop.org/archive/glu/glu-%{version}.tar.xz
+Source1:        LICENSE.PTR
+Source2:        make-git-snapshot.sh
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  libtool
+BuildRequires:  mesa-libGL-devel
+#Requires:
+Provides:       libGLU
```

Comment:
```suggestion
Provides:       libGLU = %{version}-%{release}
```

Reply:
Done.

Reply:
Are you sure? I don't see this in the latest changes.